companies/bancopichincha-es.json

```diff
@@ -0,0 +1,37 @@
+{
+    "slug": "bancopichincha-es",
+    "relevant-countries": [
+        "es"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "Banco Pichincha España S.A",
+    "address": "Calle Lagasca 4\n28001, Madrid\nEspaña",
+    "phone": "+34 914 36 25 94",
+    "email": "dpo@bancopichincha.es",
+    "sources": [
+        "https://www.bancopichincha.es/politica-privacidad/"
+    ],
+    "required-elements": [
+        {
+            "desc": "Nombre",
+            "type": "name",
+            "optional": false
+        },
+        {
+            "desc": "Dirección",
+            "type": "address",
+            "optional": false
+        },
+        {
+            "desc": "NIF/DNI",
+            "type": "input",
+            "optional": false
+        }
+    ],
+    "comments": [
+        "Debes llenar el formulario correspondiente, firmarlo, adjuntar copia del NIF/DNI y enviarlos por email o correo postal.\n\n<b>Solicitud de acceso:</b>\nhttps://media.bancopichincha.es/wp-content/uploads/RGPD_FORMULARIO_WEB_EJERCICIO_DERECHO_ACCESO_V01_BPE_NUEVO-LOGO.pdf\n\n<b>Solicitud de supresión:</b>\nhttps://media.bancopichincha.es/wp-content/uploads/RGPD_FORMULARIO_WEB_EJERCICIO_DERECHO_SUPRESIÓN_V01_BPE_NUEVO-LOGO.pdf\n\n<b>Solicitud de rectificación:</b>\nhttps://media.bancopichincha.es/wp-content/uploads/RGPD_FORMULARIO_WEB_EJERCICIO_DERECHO_RECTIFICACIÓN_V01_BPE_NUEVO-LOGO.pdf\n\n<b>Solicitud de objeción:</b>\nhttps://media.bancopichincha.es/wp-content/uploads/RGPD_FORMULARIO_WEB_EJERCICIO_DERECHO_OPOSICIÓN_V01_BPE_NUEVO-LOGO.pdf"
```

Comment:
The HTML tags won't be rendered on the website, so we should remove them.

If my translator is correct, this suggests that users have to use the linked forms, which is not correct. I would prefer if we simply list them as an option.